hack/tests/e2e-ansible.sh

```diff
@@ -133,6 +133,14 @@ then
     exit 1
 fi
 
+# Remove any "replace" and "require" lines for the SDK repo before vendoring
+# in case this is a release PR and the tag doesn't exist yet. This must be
+# done without using "go mod edit", which first parses go.mod and will error
+# if it doesn't find a tag/version/package.
+# TODO: remove SDK repo references if PR/branch is not from the main SDK repo.
+SDK_REPO="github.com/operator-framework/operator-sdk"
+sed -E -i 's|^.*'"$SDK_REPO"'.*$||g' go.mod
```

Comment:
You might be able to use `go mod edit -droprequire` and `go mod edit -dropreplace` instead. Those flags don't seem to cause tag/version/package lookups.

Reply:
I'm fairly certain any `go mod edit` command will perform lookups:
```
$ GO111MODULE=on go mod edit -droprequire github.com/operator-framework/operator-sdk
go: errors parsing go.mod:
/home/estroz/go/src/github.com/test-org/metrics-operator/go.mod:22: invalid module version "master": version must be of the form v1.2.3
```

Reply:
Ahh. I didn't test with branch names (just non-existent tag versions). Oh well.